testutil/fuse/fuse_windows.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 The qiffang Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
you still need `// +build windows` no?

Reply:
Naming a file fuse_windows.go will cause it to be included only when building it for Windows, it is the same as adding the build flag "// +build !linux,!darwin !cgo" in the file.